src/shared/PlayerData.ts

```diff
@@ -78,7 +78,7 @@ const overlayCfg = {
   type_counts: false
 };
 
-const primaryBounds = remote.screen.getPrimaryDisplay().bounds;
+const primaryBounds = remote ? remote.screen.getPrimaryDisplay().bounds : {};
```

Comment:
TS doesn't like this line now. I'm guessing it's just here for the unit test environment? Can we get away with some simple defaults?
```suggestion
const primaryBounds: Electron.Rectangle = remote
  ? remote.screen.getPrimaryDisplay().bounds
  : { width: 800, height: 600, x: 0, y: 0 };
```